packages/modelviewer.dev/data/docs.json

```diff
@@ -552,6 +552,15 @@
         "links": [
           "<a href=\"../examples/scenegraph/#transforms\">Related example</a>"
         ]
+      },
+      {
+        "name": "interpolation-decay",
+        "htmlName": "interpolationDecay",
+        "description": "Controls the rate of interpolation when the camera or model moves, due to either user interaction or attribute changes. The decay is asymptotic and the value is in milliseconds, where the majority of the movement will occur within this value's time. Doubling this value will cut the speed in half.",
+        "default": {
+          "default": "50",
+          "options": "values greater than 0"
+        }
```

Comment:
Can you also add the link section like above to point to your example?

Reply:
No problem